qemu/tests/cfg/multi_vms_with_stress.cfg

```diff
@@ -15,9 +15,12 @@
     pre_command = "mkdir -p ${stress_inst_dir}"
     post_command = "killall -9 stress; rm -rf ${stress_inst_dir}"
     ovmf:
-        smp = 1
-        vcpu_maxcpus = 1
-        vcpu_sockets = 1
-        vcpu_dies = 1
-        vcpu_cores = 1
-        vcpu_threads = 1
```

Comment:
we removed these lines, when we run testing against qemu 8.0 for example with ovmf, then there are two vms, what's more, the cpu settings are gone, we could use the default, is this expected?

Reply:
@zhencliu ,The case test point is stress. The number of CPU should not limit to 1 IMHO.  
In our past testing , using 5 VMs sometimes  failed on booting due to system is too busy , I think the testing is meaningless. so I suggest to increase CPU number and reduce VM number.
@zhenyzha ,suggest the number of VMs to  vms = "avocado-vt-vm1 vm2 vm3" ， if failed case too often, we may consider only using 2 VMs.

Reply:
Corrected

Reply:
ovmf is slower than seabios, that is why ovmf uses less vms. 
It is acceptable to use 3 Vms for both seabios and ovmf.

BTW, the CPU feature does not change before/after 8.2.0, so it does not need to distinguish the cup assignment according to qemu version. Using the new assignment (max CPU ) is enough.

set_maxcpus = yes 

if we want to test two scenarios of CPU assignment, I suggest changing the case name to with_default_cpu and with_max_cpu and not involving the qemu version.

-with_default_cpu:
-with_max_cpu:
    set_maxcpus = yes

@zhenyzha one case or two cases are both fine to me.

Reply:
> ovmf is slower than seabios, that is why ovmf uses less vms. It is acceptable to use 3 Vms for both seabios and ovmf.
> 
> BTW, the CPU feature does not change before/after 8.2.0, so it does not need to distinguish the cup assignment according to qemu version. Using the new assignment (max CPU ) is enough.
> 
> set_maxcpus = yes
> 
> if we want to test two scenarios of CPU assignment, I suggest changing the case name to with_default_cpu and with_max_cpu and not involving the qemu version.
> 
> -with_default_cpu: -with_max_cpu: set_maxcpus = yes
> 
> @zhenyzha one case or two cases are both fine to me.

Make sense, IMO, we may use the max cpus to do the performance testing